charts/ocis/values.yaml

```diff
@@ -4,7 +4,7 @@ image:
   # -- Image repository
   repository: owncloud/ocis
   # -- Image tag. Defaults to the chart's appVersion.
-  tag: ""
+  tag: "latest"
```

Comment:
On this branch. As we're using the `latest` tag I think we should also set the `pullPolicy` to `Always` (see line 11). Otherwise it's pretty easy to run with an outdated image.
I just ran into that. One of my nodes still had a very old ocis:latest still cached. `Always` should prevent that from happening.

Reply:
Makes sense :+1: changed `pullPolicy` to `Always`